### ISSUE TITLE
OnFrame error when hovering physical damage in Calcs screen

### ISSUE THREAD

**Issue by Jesperson** [PathOfBuildingCommunity/PathOfBuilding#3502]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

When hovering over the "Skill Hit Damage" amount I expect it to show me where the damage is coming from.

### What is the actual behaviour/value?

I'm getting an error:
![bild](https://user-images.githubusercontent.com/14894497/138184573-7105e20a-d405-45b2-ac09-cf089533b904.png)

https://pastebin.com/HuJ03qt5

### How to reproduce the issue

Run Templar : Guardian

1a. Have Absolution -> Added Fire Damage -> Minion Damage -> Controlled Destruction
1b. Run Hatred
1c. Run Herald of Purity
2. Run the Lightning Mastery which grants 40% conversion from phys to lightning
3. Equip Call of the Brotherhood
4. Minions have 30% of physical damage as extra chaos damage <- Add this to Custom Modifiers to simulate Unholy Might

It seems like the last step is the cause, I don't know if I'm adding it incorrectly so feel free to close this issue if that's it.

### Build code

```shell
eNrVPGtP28q2n9tfYUXa0j06JniethE9RwECpIVCCdDHl2qIJ4lbxw72JJBu9b_fNbZDHEjCxHtvXV0-QDxZ79esNZ52_7-Po8iayjQLk_hdAzWdhiXjXhKE8eBd4-b6eMdr_Pc_b_cvhRpe9A8mYaS_-c_bN_v5ZyuSUxm9a_iApkQ6kOp2Top8h7WxiNVQJvG5-JGkJ0nwrvExiWXDuhNxEKr5Uy8SWfZRjOS7xrUcjSORNiyR9WQcHC6-OZmINAhF3LBGIoy7Se-nVCdpMhmD1A1rGsqH8yQAuM755cXVdYV1GFdZg-hv9i8jMZNpVwllZfDrXaMFFhADeRoqICWiCdBB2GG4Sajjc-o6tLG7EfNIjOD3tsjdsZTBAqnJGVn8_N8jXaay3e_Lngqn8jAN1eFQxL2Kks21mNtDn08iFY6jUKYVDLYO4_QlecdZB3ydKBEdXXYXsL6PeZMR1-GIU_rP4CWLUFor2edQDQ8i8EwNLp1BHCpZRcTI9ZqMIR9T1_f4K4jLAeu5iDax43LfRz4lmwRewRgjhzUdzwOudDPyZRJmSfwXjFpD4cNkdBfGy0Y2FHmO2poOthW4lUpx0S_y50oE4SRbMHfXIZ2LWBwm2SJ0OFoHehb25RLo2ihrd83griAgzCC1mJcyhQqtzBC0sFshdNWiDnhr3XMl76uACK211pF8XJifbqBXBfS9tTkUq4o3vQ30qpAb5ZsmCvZNM1MueHtoU8nv7WnoTtyrbEzuJsI3cSozmU4re8UmFssopYNNSrJGvJIDudCXeU2M_Y3RfiZlb3gCm_-VUNIsSBeupJvtpIGrdsLeJrIrzEQ5NsN4biXqNaGgMJd73iYDaCLLFsN-c2Md2dJcn6G9MqglsUwHs-4wlJEBdG7aKkrVxNSIybLORihbKt6eiqyaeYhvVqcAr2qyPgQlbO6AEMhn3crfiHGZJj90ixZth9ZKR8kkNXRhAVxVma010uVwloU92KvzBuNKBpOeWWF7ajXPk6kcQYrkPSs09QvPrEM9iGAcMNUcyEbRVhgtpUTv51ESDORWTLbH6E7GYygUOsBM8Y7DFKychZVddQeT16EvYHA5FOPXGRwmOguXGbivAxvTPwsHQxXDVGnM5BmGuSZDkWTPuTBsAG7M4ngSRcstqudS0nQoo9TZUFl-hst4StxFcs9yHjEi7E6g9emfPIBoQz17ZwbFogJ9Lhadzvp4SWX8a2ZMfwnciEE7DiapjnRjHs8xVrGBoR9S6NUxHRPPc5vQ4PjI8Zi3Dnl5eMZN5PGnH3cd0ovpkWFMaJPDvIE86vib8VZMjy8gV06PhlxWDXHMbxKXbeL2EgvC2qdNmIQ2s1s9-m1pkW2kXTXzGcq61GD76GkCWwm43J74qLkZfHNr8gJlZbeFKGUmCKs7p40om8Tb381P_PSnzmicpCpfPBRRL8sJd-LxRFlxflaXDZOHglHDukuSSIoYSlo6kaUQS7C69H2PJ6M7ffRT_F1s9VXIUZj1vt9N-n190NcA8dP8kLJ9fNw-vO7ctkuUrswbDquXRJEYZzJ41-iLKAOMMCgrbRe6rJ4ygj8NVVk0TKChie6lIVQxE2Cd5EYi5-XHBFIfpBkBggdFJM0Ndj0bSx0zmanNyu7FBDovYUaQRQUxUzCvVGZukD0xM4J8Sg8j6HYkW2GkG1kzq51DeBc9sBk8dKVpeDdRhtGWlzUjOfTcbKRgtToZZocZYDmcGAlRTm8msGXrZxaUMB8Yu-1I9iWEu2Ha5-XkWvyUcVlY50V0P8-1zMqgup7IUXYwg-3rWJv32Ya2BFBWViuQfTGJ9PqniYhCNdOVu7J6VryqwbCo67MeNQoyOrkzMPrZWfFNK1IlBc1jLnuhmRYwfwHTyke1_DkXP38LE8a9aBLITly2wXPRInGnWTesgX5hc5hMYth44jDS75p0oxvMAZ9R1kQ13zf7IEkJexIldyLCTybN949yV6tIg5e-KRdbd1kSTbRvyvdM7yeBbgejhWmqTNBcrJxSJ6gSeMk453lfGK6T15Xc8qC0HOnnc6lEIJTY7Sgw66627W6OCJ-qdO8r3ntmnl5hOdLId0Q9mq8Vaa2yu68Z1NgYZQS1gkAGeq6cN9fb2uApENdR2mAKtGQKjW9pAlZ1z_47lS2M-5c1XUnGVM0C-R9T8TCJVQqVC2qNhB5rfnxTW9fN9EyVXlCxqmSKglvkUb0KNbfSP1iijD1wKlSqJ82aVeQZuqlpS7T_F6ZcU-3RNkaWqYiCi_7lBLrl2ao6jrb2wHOar3uALnkgR7eSvrVCKKhrj2HWBnqym0RBKNM5t_wc7Lnfyk95P3GdSmmJwsI5I1Ru4_BQvWGh1SLaH5mS6axsQt81_sTUsRlxKP9t_0lhbGY2p8RD-slFvmsTeCLwRHyKuE0dx9VPiDKH2QBNqX7CCMOTyynO8TDFtu-5_DcYAOY4YOMSz7OJ71Fku5RSG_nIITZzKbG5Awg2ZRwAOCfUpjC7ExtxlzJbS0c9wrGNXQwYICEAc0Q0iuMgx8aceghWGEjAfMLsUhgfM2oT7BMMKiDNkjoYALkHVDnyuY0QxUCduFovSrgDojBAQszJKVLkA5JPuM0o90Euz0UgAeGeb7s-8WwgDoBIy0WQ5wJdRoAdJg527cKWGPTktuv5nl2Ykzq-zTBwx9TDuVAMbMwwcEeYuD7gua6Nkc8AA8Qj1PMcLRIFgX3mILt0BAZNYJ1w0Nj1tGbE51o17IJRsAMm4y4C5cGBzCYgAWjjEAS0CQG1cycRps2BHN-hNijD4RdggxtcTxvZARmIjgEfjGzDMwhPwBRaSuQ7tn6hpWV1qc1ABM9mDIGchLlaNeAImmAfBKEYPKWPZkAmTsFOHgiCHC0OIxi0JpjAOjjKBQFdjhBAu762v44zMIl2LEgBWAwhF0xIQTdYgXVwDsO2yxjTznFtH2s-DkQbKA_LDQtqn6xcWkK8vI-kc4KV9agdQPIFH3W45gvVFUvCxyTtSp3iCBFI7LCXZ22qdvGR_p0nZAcWs90xUIZszHZvhYjysUE_NoMgK9KhuOt0OEkzaRV5WKxraXTQlBXyzRv2hy7AqRQwXpSQun7MYG6ycvSsEHR3IWmhy-5zZfZvrs4KokOlxtne7u7Dw0NzLNQw6cvHMJLNXjKaC76TF6YdbbTdFvwctG6OWge_eiPeDo7b9_3j0ffs4uD064_H-yxyjsety0OPj8ZHyeBoOn3_-OEbaX85m_nBw4lojTu3g4_ZjTj9_O3m7P3s_vED6ridhHP5Pb2IftyOU_b-_v5nvxWfHR-cffQG-Mz53L8YPpKT2am6p1-uug9fLweYfx1558fZp5vbLyct0e18OkpPrg_xt50PIrts3R7H_Tj4Nmafz4YJHYwnX68_fTq9QR9omKjow3jw8z7sih_O3bR_0FajSWvmfDu4TS6-ysPjL58pTr-IZHJ-ffez41-m3fZXKu5OruP3s--fTgnLhl9Uirn7_eJDdHzlzNjFAzmftme_FB4Eg8f25fu2m7rJ5V1h-rml94vra_NgKp4WfuaQLhBFqthvypO7FXAIUt8AjkPJpCZwFFLPAA7S1jehRz3u-ib0CEIGYD51TKyS7xMmXH2ogCZwUKuM6HEXMxP5oOyYWJkgbuQNKJHcBE7fkzKJFt_1XZPgg33ShC2HnsDELBx2RpOY4qCGCZwDHYAJHOyXvpE3HCO2sNVzIzWIb5RqECwm9Bi0BCZWhpbFiK_e303oYY-6Rt5F2POMkgi2bSN6nBvZ2Xc8kzBgVF-MMkleaFxM7OxBt2EC52KzcNa3g0zoOdAyGdmPGvHF2KUmfoO-y_eM8gMbFStMfdcxyg-z4kx9bgYH6Wu09WL9XtBIDxMwRJCRuoRwYkLP9bkROe4bkeMwqJiQo3TVjgUD6aLbgQeYPfMxVQ-n-sPHRBWdqF6cP-Sj620oH6xMirQ37KpUnyj8SpLRV-Dk4abLXJhBEPK0Y_Wx9akU6lyMn44KNGx55k3KI28l1FEIvXKan9bPJ3IN-AW05E2q5x8KOwcu7_Xu5zN-OULrz11ZnJ5MMtmV0NEHn6UYJ3G-XDkn16DWVKSh0LM-Lg7-y979SugRf8-6-dj5dNN-eyiiSHfuaiitgzSBP-kwSYK31w_JTlclsbSuwngAYEpEs2x-Mr9nYeftbcFgz7pMpYWbvFlZghkAtFRvi1edMCM8fYOfk86NdCXvc5qd0TgKe6HK9iz09k8lBtneD_kgo0ims-9p_gJFv9n7_Wcq4oHcQ7___T8I7yD-rz8slVj6-o8l4sB6uqJjXT3hZC_oifkrrCVybAezf2lqnVgBYDjQ3nqBC3rpd2siWuCWqNXJaCFHcVxakhmJWFTwiLNDnSU8_RLMyt-gy1TkL3P0K_FNQpTeBnLM-UM79DlrME48lan2xdxSSzJtpIp_022pftWjYKZvlWXWUEylBTP1H1YvfzGrYbtDSEtLDEQYZ8o6TpNfMrZA31FYzI7750lwpS1k5XYqzvyCp1fza7_Hr3xPXvl-fjs7P18rjquiZH4HQAesValZT_cEitzckJfVFKxSLAAt1H0QY6t1N8syEVllZVvR2qxCfY6GzdA2Qx0kicpqCrRKl1eEOpCR2hLlOBLZT4sYiYRXiUTqmdfMCPhvQqP1hOT1uPH61qwTT6944FRGI6m2ZbQikngdPq-E30mUTPV9we0Jk_pZhOrwo_ViAdWLPPq6e0wMuy3ZVeFVU3GjusNqyFOjBpLX2AQzq7iusqX7Vlu5VnyxOkhGKWniiFp6stpmxTX8zutlEqkXvmT7KKP1y36NmEa1jU9MAoKaANUQ-y80P4aerO8FXiMq67vhL4SLaUnWuIZZYw5aw3J5w4_rlBhc27w1y4W5P-uoQwzs9HIwMnYDrW0tVkcbk3ypH-SviNQaTaIVe1sxccLImB9P5Sc_-RX3JO6Hgxe30XuTTCUjmGGzxVX04sJGOW6TYmAfl_8ezgqKeV1klnxUqdCTeJKVq_OLoCWr_d3n_9_G_wK99Ypr
```

**Comment by Wires77** [comment]
This seems to be fixed in the dev branch, and will be fixed on our next release

**Comment by LocalIdentity** [comment]
Fixed in 2.12
